src/main.cpp

```diff
@@ -55,23 +57,43 @@ void loop() {
   Serial.print(moisture);
   Serial.println("%");
 
-  // Write to ThingSpeak. There are up to 8 fields in a channel, allowing you to store up to 8 different
-  // pieces of information in a channel.  Here, we write to field 1.
+  // push moisture data to thingspeak dashboard
   int x = ThingSpeak.writeField(myChannelNumber, 1, moisture, myWriteAPIKey);
   if(x == 200){
     Serial.println("[+] Channel update successful.");
+    delay(500);
   }
   else{
     Serial.println("[*] Problem updating channel. HTTP error code " + String(x));
   }
-// BMP280 Sensor Code
+// poll sensor data
   Serial.print("Temperature = ");
   Serial.print(bmp.readTemperature());
   Serial.println(" *C");
 
+  // push temperature data to thingspeak dashboard
+  x = ThingSpeak.writeField(myChannelNumber, 2, moisture, myWriteAPIKey);
```

Comment:
change the sensor value variable to temperature
```suggestion
  x = ThingSpeak.writeField(myChannelNumber, 2, temperature, myWriteAPIKey);
```